airflow/providers/jenkins/operators/jenkins_job_trigger.py

```diff
@@ -197,7 +200,7 @@ def execute(self, context):
         jenkins_server = self.get_hook().get_jenkins_server()
         jenkins_response = self.build_job(jenkins_server)
         build_number = self.poll_job_in_queue(
-            jenkins_response['headers']['Location'], jenkins_server)
+            jenkins_response['headers']['Location'], jenkins_server)  # type: ignore
```

Comment:
I can't seem to get this one either. It keeps saying this, but I can't tell which one it wants me to change:

```
airflow/providers/jenkins/operators/jenkins_job_trigger.py:204: error: Invalid index type "str" fo
"Union[str, Mapping[str, str]]"; expected type "Union[int, slice]"
                jenkins_response['headers']['Location'], jenkins_server)
                                            ^
```

Reply:
Handled this one with a simpler type check.